### DISCUSSION TITLE
recuperar un proyecto eliminado hace 4 meses

### DISCUSSION THREAD

**Discussion by caracaicedo** [community/community#108926] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Buenos dias , como puedo recuperar un proyecto que elimine hace 4 meses y necesito restaurarlo de nuevo ?

**Comment by CoreTiQ** [comment]
Navigate to Your Repository: Once logged in, navigate to the repository page from which you want to recover the project.
Search Version Control History: Search through the version control history of the repository for the version you want to recover. You can do this by going to the "Commits" or "History" tab on GitHub.
Recover the Deleted Version: Once you find the deleted version you want to recover, you can usually view the changes and revert them. You might need to use the "git revert" command if you're using Git on your local machine.

**Comment by caracaicedo** [comment]
pero ten en cuenta que el repositorio no existe yo lo elimine hace 4 meses y ahora necesito recuperarlo de neuvo y nop veo la opcion que me indicas

**Reply by CoreTiQ** [reply]
![image](https://github.com/community/community/assets/160239766/62e251a8-044b-41d7-be27-1a393b8119e5)
